tests/modules/testrdb.c

```diff
@@ -305,7 +317,7 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
         testrdb_type = RedisModule_CreateDataType(ctx, "test__rdb", 1, &datatype_methods);
         if (testrdb_type == NULL)
             return REDISMODULE_ERR;
-    } else {
+    } else if (conf_aux_count==1 || conf_aux_count==2 || conf_aux_count==5 || conf_aux_count==6) {
```

Comment:
is there some easy to follow table that we can document as to what each number stands for?
i.e what each bit does?

Reply:
I changed it to use flags, I hope it is more clear now.